input/postgres/relation_stats.go

```diff
@@ -20,7 +20,16 @@ locked_relids_with_parents AS (
 	UNION SELECT relid FROM locked_relids
 )
 SELECT c.oid,
-			 COALESCE(pg_catalog.pg_table_size(c.oid), 0) AS size_bytes,
+			 COALESCE(pg_catalog.pg_table_size(c.oid), 0) +
+			 COALESCE((
+				SELECT pg_catalog.sum(pg_catalog.pg_table_size(inhrelid))
+				FROM pg_catalog.pg_inherits
+				JOIN pg_class t ON inhrelid = t.oid
+				WHERE inhparent = c.oid
+					-- Only include sizes from child partitions skipped by ignore_schema_regexp.
+					-- Child partitions tracked by the collector have their sizes added by mergePartitionSizes.
+					AND ($1 != '' AND (n.nspname || '.' || t.relname) ~* $1)
+			 ), 0) AS size_bytes,
```

Comment:
The screenshot below is from local testing. 11:40 shows the detailed stats from #635. I then updated the collector config with `ignore_schema_regexp = 35d_`, and then 12:10 shows the updated behavior with this PR. Ignore the in-between timestamps where I was fixing bugs.

![Screenshot 2024-12-19 at 12 12 08 PM](https://github.com/user-attachments/assets/349a8c31-457c-4a49-bec5-15523b72c90e)